python/pyspark/sql/session.py

```diff
@@ -1662,7 +1662,7 @@ def sql(
 
         And substitute named parameters with the `:` prefix by SQL literals.
 
-        >>> from pyspark.sql.functions import create_map
+        >>> from pyspark.sql.functions import create_map, lit
```

Comment:
Out of curiosity, why is the missing import not detected? Are we skipping the doctest?

Reply:
oh, `lit` was import at the beginning of this file.
but for the parity doctest, it is missing, so add it here

Reply:
Thanks!